lua/refactoring/tests/refactor/123/go/multiple-vars/inline_var.expected.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import "math"
+
+type Order struct {
+	quantity  float64
+	itemPrice float64
+}
+
+func orderCalculation(order Order) float64 {
+	basePrice, i := order.quantity*order.itemPrice, 3
```

Comment:
Shouldn't basePrice (and then then the first identifier) be removed? Then this should be left:
`i := 3`

Reply:
Unfortunately this will vary by language, but I think we need this to properly inline when multiple declarations are present.

Reply:
Ah just saw comment in header, we can make that another ticket but might want to print something to buffer saying (please remove initial declaration, feature coming soon).

Reply:
Yeah the logic will be different for each one so I've just left it as is - do you think that this should be done in this PR or another. I think to do it dynamically it will take new codegen and logic to quite literally reconstruct the declarations (it will also vary depending on how many identifiers in the declaration). Do you think I should do it in this PR or another one @teddylear? It still takes it out for the one declaration, by the way.

Reply:
Im ok with leaving as is for multiple declarations and doing in another PR. But we might just put a print statement saying like "Removing initial declaration on multi declaration lines not supported yet" so users aren't confused why its thr declaration is still there.

Reply:
Sounds good. Thanks for the review!